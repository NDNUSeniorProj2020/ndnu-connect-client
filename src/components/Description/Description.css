.description-container {
	background-color: #21285C;
    color: #ffffff;
    height: 100%;
    width: 50%;
    position: absolute;
    left: 0;
    overflow: hidden;

}

.header{
    color: #ffffff;
}

h1 {
margin-left: 20px;
}

p {
   margin-left: 20px;
   margin-right: 20px;

}
