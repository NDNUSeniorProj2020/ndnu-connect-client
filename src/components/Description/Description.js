import React, { Component } from 'react';
import './Description.css';
export default class Description extends Component {
  render() {
    return (
      <view style={{flex:1, flexDirection:'row'}}>
      
      <div className="description description-container">
        { this.props.children }
        <h1 className="header">Welcome to NDNU Connect</h1>
        <div className= "info-container">
          <p>NDNU Connect is a tool for students to connect with tutors as well as NDNU's Alumni</p>
        </div>
      </div>
      </view>
    )
  }
}
