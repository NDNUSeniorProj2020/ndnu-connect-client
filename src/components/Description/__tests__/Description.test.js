import React from 'react';
import renderer from 'react-test-renderer';

import Description from '../Description';

describe('Testing Description component', () => {
	describe('Snapshot tests', () => {
		it('should render without crashing', () => {
			const tree = renderer.create(<Description />);
			expect(tree).toMatchSnapshot();
		});
	});
});