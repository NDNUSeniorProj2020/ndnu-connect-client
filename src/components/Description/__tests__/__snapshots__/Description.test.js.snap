// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Description component Snapshot tests should render without crashing 1`] = `
<div
  className="description description-container"
>
  <h1
    className="header"
  >
    Welcome to NDNU Connect
  </h1>
  <div
    className="info-container"
  >
    <p>
        Notre Dame de Namur University (NDNU) Connect is an application that connects students, tutors and alumni on one platform. NDNU Connect is a React-Django web application that allows past and present students of the university to connect and assist each other in their academic journeys and present career opportunities. It also serves as a one stop shop in order to help place students into internships and to help current students with career guidance.
    </p>
  </div>
</div>
`;
